File: wrappers/mvc/demos/Kendo.Mvc.Examples/Models/Chart/ProteinScoreItem.cs
namespace Kendo.Mvc.Examples.Models
{
    public class ProteinScoreItem
    {
        public string Name { get; set; }
        public string Abbr { get; set; }
        public int Score { get; set; }

        public ProteinScoreItem(string name, string abbr, int score)
        {
            Name = name;
            Abbr = abbr;
            Score = score;
        }
    }
}

File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/models/ProteinScoreItem.java
package com.kendoui.spring.models;

public class ProteinScoreItem {
    private String name;
    private String abbr;
    private int score;
    
    public ProteinScoreItem(String name, String abbr, int score) {
        setName(name);
        setAbbr(abbr);
        setScore(score);
    }
    
    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
    
    public String getAbbr() {
        return abbr;
    }

    public void setAbbr(String abbr) {
        this.abbr = abbr;
    }
    
    public int getScore() {
        return score;
    }

    public void setScore(int score) {
        this.score = score;
    }
}
